topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -192,3 +206,34 @@ def add_datetime_property(self, item: Item, asset_metadata: Dict[str, str]):
                 item.add_error(msg="Invalid date", cause=self.name, e=e)
         else:
             item.add_error(msg="No date found", cause=self.name, e=Exception(f"item date has no value"))
+
+    def add_centroid(self, item: Item, asset_metadata: Dict[str, str]):
+
+        centroid = {
+            "lat": string_to_number(asset_metadata.get("photocentre_lat", None)),
+            "lon": string_to_number(asset_metadata.get("photocentre_lon", None)),
+        }
+        if self.is_valid_centroid(item, centroid):
+            item.properties["proj:centroid"] = centroid
+            item.add_extension(stac.StacExtensions.projection.value)
```

Comment:
The projection extension should always be added to the stac_extensions, as the historical imagery extension requires it in stac_extensions (proj:epsg is a required field).

Reply:
I think it is always added when adding proj:epsg